CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+# Contributing to Lookit
+
+We'd love for you to contribute to our source code and make Lookit even better than it is today!
+Here are some guidelines we'd like you to follow:
+
+* [Code of Conduct](#code-of-conduct)
+* [Requests](#requests)
+* [Developer Guidelines](#developer-guidelines)
+    * [Local Development](#local-development)
+    * [Source Control](#source-control)
+    * [Localization](#localization)
+    * [Releases](#releases)
+* [Contact Us](#contact)
+
+## Code of Conduct
+
+As contributors and maintainers of the Lookit project, we pledge to respect everyone who contributes by posting 
+issues, updating documentation, submitting pull requests, and providing feedback in comments.
+
+Communication through any of Lookit's channels (Slack and Github) must be constructive and never resort 
+to personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+
+We promise to extend courtesy and respect to everyone involved in this project regardless of gender, 
+gender identity, sexual orientation, disability, age, race, ethnicity, religion, or level of experience. 
+We expect anyone contributing to the Lookit project to do the same.
+
+If any member of the community violates this code of conduct, the maintainers of the Lookit project may 
+take action, removing issues, comments, and PRs or blocking accounts as deemed appropriate.
+
+## <a name="requests"></a> Questions, Bugs, and Features
+
+### Got a Question or Problem?
+Do not open issues for general support questions as we want to keep GitHub issues for bug reports
+and feature requests. We have an existing process for diagnosing/triaging issues and generating new
+feature requests from user input, and it starts with [Slack][slack].
```

Comment:
can we link to https://lookit.readthedocs.io/en/develop/researchers-start-here.html#a-join-the-slack-workspace either instead or in addition? (Someone not yet in the Slack workspace can't join just from the link here.) Or if you think we should have a more prominent section of the docs just labeled Slack w/ the invite link, workspace link, & explanation of how to use it we can add that concurrently

Reply:
Changing the language to place a little more emphasis on the Slack, let me know if this works for you.